nservicebus/azure-service-bus/index.md

```diff
@@ -11,37 +11,37 @@ related:
- * The main disadvantage of ASB is its dependency on TCP (for low latency), which may require opening some outbound ports on the firewall. Additionally, in some systems the price may get high ($1 per million messages).
+ * The main advantage of ASB is that it offers a highly reliable and low latency remote messaging infrastructure. A single message can be up to 256 KB in size, and a queue can store many messages at once, up to 5 GB size in total. Furthermore, it is capable of emulating local transactions using its queue [peek-lock mechanism](https://msdn.microsoft.com/en-us/library/azure/hh780722.aspx). 
+ * The main disadvantage of ASB is its dependency on TCP (for low latency), which may require opening outbound ports on the firewall. Additionally, in some systems the price for the service may get high ($1 per million messages).
 
-Note: Publish/Subscribe and Timeouts (including message deferral) are supported natively by the ASB transport and do not use persistence.
+NOTE: As part of the Azure support for NServiceBus, one can choose between two transports provided by the Azure platform: Azure Storage Queues and Azure Service Bus. Each of these two options has separate features, capabilities, and usage characteristics. A detailed comparison and discussion of when to select which is beyond the scope of this document. To help decide which option best suits the application's needs, review the Azure article [Azure Queues and Azure Service Bus Queues - Compared and Contrasted](https://azure.microsoft.com/en-us/documentation/articles/service-bus-azure-and-service-bus-queues-compared-contrasted/).
 
 
 ## Enabling the Transport
 
-Firstly, choose Standard or Premium Messaging Tier for Azure Service Bus when creating the namespace at Azure portal.
+When creating the namespace at the Azure portal, choose Standard or Premium Messaging Tier for Azure Service Bus. 
 
-Secondly, reference `NServiceBus.Azure.Transports.WindowsAzureServiceBus` NuGet package.
+In the solution reference `NServiceBus.Azure.Transports.WindowsAzureServiceBus` NuGet package.
 
 ```
 PM> Install-Package NServiceBus.Azure.Transports.WindowsAzureServiceBus
 ```
 
-Then use the Configuration API to set up NServiceBus, by specifying `.UseTransport<T>()` to override the default transport:
+Then use the Configuration API to set up NServiceBus, by specifying `.UseTransport<AzureServiceBus>()` to override the default transport:
 
 snippet:AzureServiceBusTransportWithAzure
 
 
 ## Setting the Connection String
 
-The default way to set the connection string is using the .NET provided `connectionStrings` configuration section in app.config or web.config, with the name `NServicebus\Transport`:
+The default way to set the connection string is using the .NET provided `connectionStrings` configuration section in an `app.config` or a `web.config` file, with the name `NServicebus\Transport`:
```

Comment:
If we're moving away from the xml based config, should we promote app.config is the default way? Doesn't sound right.